tests/Repository/SubmitRepositoryTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Starfleet Project.
+ *
+ * (c) Starfleet <msantostefano@jolicode.com>
+ *
+ * For the full copyright and license information,
+ * please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace App\Tests\Repository;
+
+use App\Repository\SubmitRepository;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+class SubmitRepositoryTest extends KernelTestCase
+{
+    public function testUpdateDoneSubmitsWork()
+    {
+        static::bootKernel();
+
+        $submitRepository = static::$container->get(SubmitRepository::class);
+        $submitRepository->updateDoneSubmits();
+
+        self::assertSame(0, \count($submitRepository->getDoneSubmits()));
```

Comment:
You are hydrating all the DB, for just a count. Your DB can return the value directly. 

And the method `getDoneSubmits` is not used in the app, so you can inline the query here

You really must remember this rule: 

> You must not hydrate data when it's not really necessary ! It kills 🐻

 😄